### DISCUSSION TITLE
Are statistics grouped by keymap layout ?

### DISCUSSION THREAD

**Discussion by 555wolfman** [monkeytypegame/monkeytype#1852] [Questions] [answered]
I am fast at QWERTZ layout and going to learn a new, ergonomic layout called Bone.

Due to i am beginning with Bone my wpm will start from scratch.  After a time i want to control and keep my QWERTZ skills.
Therefore it would be helpful if statistics are grouped by the keymap layout.

Is that the case?  
Or is the best workaround to create a new user for those different skills?

Olaf

**Comment by frogfrogfrogfrogfrog** [comment, marked as answer]
This isn't done automatically. If you want to have different statistics with different keymap layouts you need to create a tag (located at the top of settings or you can do so through the command line) and create a tag. Once you've created a tag you can do typing tests on that tag which will keep those scores. If you want to do it on a different keymap then create a new tag and learn it on that one.

**Reply by 555wolfman** [reply]
Works like a charm. Thank you!
